### PR TITLE
Switch to stockcharts

### PR DESCRIPTION
**Pull request by monicachew**
You can check this out at http://people.mozilla.org/~mchew/test_pinning_dashboard/

### CODE REVIEW

**Comment by monicachew** [comment]
For some reason, the event markers don't always show up. I think there's mouse contention with the tooltip.

**Comment by monicachew** [comment]
@mozkeeler ready for review. I gave up and used human-readable titles on the markers.

File: index.html [hunk 1]
```diff
@@ -2,8 +2,12 @@
 <head>
 <script src="http://telemetry.mozilla.org/v1/telemetry.js"></script>
 <script src="http://ajax.googleapis.com/ajax/libs/jquery/1.8.2/jquery.min.js"></script>
+<!--
```
Comment (mozkeeler) [inline index.html]: Should we just remove these instead of commenting out?

**Comment by mozkeeler** [comment]
That's my only concern. Otherwise, looks great!

File: index.html (hunk 1 above)
Comment (monicachew) [inline index.html]: Sure, done!

**Pull request merged**